torchvision/models/mobilenetv3.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, input_channels: int, squeeze_factor: int = 4):
         self.relu = self.activation
         delattr(self, "activation")
         warnings.warn(
-            "This SqueezeExcitation class is deprecated and will be removed in future versions. "
+            "This SqueezeExcitation class is deprecated since 0.12 and will be removed in 0.14. "
```

Comment:
Coz we have alternative Squeeze Excite in ops.  Can we suggest to use it instead. 
Like

Mobilenet Squeeze Excite is deprecated since 0.12 and will be removed in 0.14. Use ops.SqueezeExcite instead.